### DISCUSSION TITLE
Detect holes boundary in pointcloud data?

### DISCUSSION THREAD

**Discussion by K-M-Ibrahim-Khalilullah** [pyvista/pyvista#1787] [Q&A]
How can we detect hole's boundary in point cloud data using pyvista?

**Comment by banesullivan** [comment]
Can you provide some example data or further context?

Point clouds can't have holes (only a surface mesh/grid could have a hole). Are you trying to find holes in a surface mesh?

**Reply by banesullivan** [reply]
FYI, the response in https://github.com/pyvista/pyvista/issues/1785#issuecomment-962672896
